fmriprep/workflows/bold/resampling.py

```diff
@@ -86,12 +81,16 @@ def init_bold_surf_wf(
 """.format(out_spaces=', '.join(['*%s*' % s for s in surface_spaces]))
 
     inputnode = pe.Node(
-        niu.IdentityInterface(fields=['source_file', 't1w_preproc', 'subject_id', 'subjects_dir',
+        niu.IdentityInterface(fields=['source_file', 'subject_id', 'subjects_dir',
                                       't1w2fsnative_xfm']),
         name='inputnode')
     itersource = pe.Node(niu.IdentityInterface(fields=['target']), name='itersource')
     itersource.iterables = [('target', surface_spaces)]
 
+    get_fsnative = pe.Node(FreeSurferSource(), name='get_fsnative',
+                           run_without_submitting=True)
+    tonii = pe.Node(fs.MRIConvert(out_type='niigz'), name='tonii')
```

Comment:
If all you need from the T1.mgz is the shape/affine, then you don't need to convert to NIfTI. Nibabel supports MGH files just fine.

Reply:
I wasn't convinced about nitransforms' utilization of nibabel... I'll give your suggestion a try

Reply:
okay, nicely works - pretty slick.